numpy/_core/tests/test_shape_base.py

```diff
@@ -383,6 +383,11 @@ def test_concatenate(self):
         assert_(out is rout)
         assert_equal(res, rout)
 
+    def test_concatenate_same_value(self):
+        r4 = list(range(4))
+        with pytest.raises(ValueError):
```

Comment:
Maybe add a `match=...` so that it is clear which `ValueError` you are expecting.

Reply:
done